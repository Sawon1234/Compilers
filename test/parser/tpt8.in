fun foo(y : Integer) : Integer = y * 2;



a := 640802120;