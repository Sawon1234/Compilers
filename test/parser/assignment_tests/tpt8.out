fun foo < > ( y : Integer < > ) : Integer < > return y . times < > ( 2 ) ; a := 640802120 ;